lib/stdlib/src/timer.erl

```diff
@@ -130,8 +148,21 @@ apply_interval(Time, M, F, A) ->
       Message :: term(),
       TRef :: tref(),
       Reason :: term().
-send_interval(Time, Pid, Message) ->
-    req(apply_interval, {Time, Pid, {?MODULE, send, [Pid, Message]}}).
+send_interval(Time, Pid, Message)
+  when ?valid_time(Time),
+       is_pid(Pid) ->
+    req(apply_interval, {Time, Pid, {?MODULE, send, [Pid, Message]}});
+send_interval(Time, RegName, Message)
+  when ?valid_time(Time),
+       is_atom(RegName) ->
+    case get_pid(RegName) of
+	Pid when is_pid(Pid) ->
```

Comment:
Ah, please ignore me. We were calling `get_pid` early before as well, the only difference is that it was happening on the server instead of the client. :)

Reply:
Right =^^= But anyway, the general plan was to not have changes in outside behavior, so if you spot anything that may have an impact there, please report :)